.github/workflows/canary-integration-test.yml

```diff
@@ -73,8 +73,19 @@ jobs:
           toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
           timeout 15 sh -c "until kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr|grep -Eosq \"(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\" ; do sleep 1 && echo 'waiting for the manager IP to be available'; done"
           mgr_raw=$(kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr)
+
           timeout 60 sh -c "until kubectl -n rook-ceph exec $toolbox -- curl --silent --show-error ${mgr_raw%%:*}:9283; do echo 'waiting for mgr prometheus exporter to be ready' && sleep 1; done"
 
+      - name: check s5cmd version in the toolbox image
```

Comment:
Need to remove this section of the test since we moved it down to line 319. I see the new test in that section is passing, but this section is failing since we know the toolbox-operator-image is not running.

Reply:
I tried something different now, since I saw the attempt at a diagnostic output of the s5cmd version failing because it could not find the binary ...

Reply:
We need to remove this whole section, lines 79-88. This CI test runs the different toolbox. The other CI test does run the toolbox operator image so we only can test it on lines 319-328.

Reply:
@travisn  wrote:
> We need to remove this whole section, lines 79-88. This CI test runs the different toolbox. The other CI test does run the toolbox operator image so we only can test it on lines 319-328.


got it,  thanks.  removed.